donate/templates/fragments/language_switcher.html

```diff
@@ -0,0 +1,23 @@
+{% load i18n %}
+
+<form action="{% url 'set_language' %}" method="post">{% csrf_token %}
+  <input name="next" type="hidden" value="{{ request.get_full_path }}">
+  <div class="footer__language-switcher">
+    <label class="heading heading--secondary footer__language-label" for="language-switcher">
+      <svg class="language-icon" width="24" height="24">
+          <use xlink:href="#globe"></use>
+      </svg>
+      {% trans "Language" %}
+    </label>
+    <select name="language" id="language-switcher" class="form-control" onchange="this.form.submit()">
+      {% get_current_language as LANGUAGE_CODE %}
+      {% get_available_languages as LANGUAGES %}
+      {% for CODE, NAME in LANGUAGES %}
+        {% get_language_info for CODE as lang %}
+        <option value="{{ CODE }}"{% if CODE == LANGUAGE_CODE %} selected{% endif %}>
+          {{ lang.name_local | capfirst }}
```

Comment:
just because I'm curious: is `capfirst` unicode aware?

Reply:
The documentation doesn’t provide much detail, but from what I’ve seen, it seems so, yes — it’s smart enough to ignore cases where the concept of capital letter doesn’t exist, and does the right thing for cases like é → É